python/ray/actor.py

```diff
@@ -413,7 +413,8 @@ def _remote(self,
                 name=None,
                 detached=False,
                 placement_group_id=None,
-                placement_group_bundle_index=None):
+                # TODO(ekl) set default to -1 once we support -1 as "any index"
+                placement_group_bundle_index=0):
```

Comment:
But the index of bundle is begin from 0? Will this be ambiguous？

Reply:
I'll change it to -1 soon, this is just a temporary value.